src/classes/window/presenter.vala

```diff
@@ -153,7 +184,83 @@ namespace pdfpc.Window {
          **/
         protected int next_allocated_width;
 
-        /**
+        protected bool on_button_press(Gtk.Widget pbut, Gdk.EventButton event) {
+	    if (event.button == 1 ) {
+                var w = this.get_window();
+
```

Comment:
This looks unaligned.

Reply:
Seems to me OK. There are tab characters (probably copy&paste'd from another function), but there are many other places in the pdfpc sources. Maybe we need to run everything through expand once?